src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -194,7 +196,28 @@ export default class AddStaffNotationDialog extends Mixins(DateMixin, EnumMixin)
       this.saving = false
       return
     }
-
+    const dataSpGp = {
```

Comment:
instead of making two-variable data and dataSpGp, can combine and push to object 
as discussed you can look here for example
https://github.com/bcgov/business-create-ui/blob/main/src/mixins/filing-template-mixin.ts#L495

Reply:
done @severinbeauvais please review again when you have the chance

Reply:
This is Faiz's conversation to resolve.